.github/CONTRIBUTING.md

```diff
@@ -22,6 +26,23 @@ The long version:
 
 *Please* provide feedback about how these practices impact your ability to contribute. You might voice that feedback on the [Apereo licensing discussion Google Group][] or in any other way in which you are comfortable.
 
+## Conventional Commits
+
+This project uses [Conventional Commits][].
+
+Conventional Commits compliance is not a hard requirement to contribute. Do
+what you can. If necessary commit message style can be further worked up at the
+Pull Request / code review layer. Non-compliant commit messages will be flagged
+by continuous integration (e.g. Travis-CI) on pull requests.
+
+You can locally lint (check for style compliance) your most recent commit message by `npm run lint-commit`.
+
+You can locally lint your commit messages via a pre-commit hook by
+
+1. Setting `"commitmsg": "commitlint -e",` in your local `package.json`
```

Comment:
Is this still needed since this is included in the `package.json` already?

Reply:
Yes? `commitmsg` does not appear in [the package.json](https://github.com/uPortal-Project/uportal-app-framework/blob/master/package.json) as of https://github.com/uPortal-Project/uportal-app-framework/pull/581/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L6 's removing it?

Reply:
Oh, I see. I'd inadvertently re-added this to `package.json` in this changeset. Hasn't meant to include that change; fixed.